src/VisualStudio/Core/Def/Implementation/FindReferences/Contexts/AbstractTableDataSourceFindUsagesContext.cs

```diff
@@ -481,9 +495,77 @@ private void UpdateCustomColumnsVisibility(ImmutableDictionary<string, Immutable
                 }
             }
 
+            private void UpdateCustomColumnVisibility(string key)
+            {
+                // Check if we have any custom reference data to display.
+                // columnDefinitionManager will be null under unit test
+                var columnDefinitionManager = TableControl.ColumnDefinitionManager;
+                if (columnDefinitionManager == null)
+                {
+                    return;
+                }
+
+                // Get the new column states corresponding to the custom columns to display for custom data.
+                var newColumnStates = ArrayBuilder<ColumnState2>.GetInstance();
```

Comment:
I believe `ArrayBuilder` was being marked as `IDisposable`, has that change made it in yet? If so, you can do using instead of try/catch